src/pages/library/DataBrowser.js

```diff
@@ -55,17 +56,26 @@ const extractCatalogFilters = dataCatalog => {
         div([snapshotAccessTypes[accessKey]])
       ])]
     }, _.keys(snapshotAccessTypes)))
+  }, {
+    name: 'Data use policy',
+    labels: releasePolicyArray,
+    labelDisplays: _.zipObject(releasePolicyArray, _.map(releasePolicy => {
```

Comment:
If you don't want it to be a flow, could you destructure the props used in `releasePolicy`?
```suggestion
    labelDisplays: _.zipObject(releasePolicyArray, _.map(({label, tag, desc})=> {
```

Reply:
Will do